lib/core/elastic_connector_actions.rb

```diff
@@ -70,9 +70,6 @@ def native_connectors
           offset += DEFAULT_PAGE_SIZE
         end
         result
-      rescue StandardError => e
-        Utility::ExceptionTracking.log_exception(e, 'Error while getting native connectors')
-        raise Utility::ClientError.new(e)
```

Comment:
On the off chance you want to keep this, I think you can also add something like `Utility::ExceptionTracking.log_exception(e, e.backtrace)` to get the stack trace.
https://ruby-doc.org/core-2.6/Exception.html#method-i-backtrace

Reply:
We can indeed, but if it's not caught I believe the caller will do it here: https://github.com/elastic/connectors-ruby/blob/main/lib/core/native_scheduler.rb